applications/StructuralMechanicsApplication/tests/test_rve_analytic.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import print_function, absolute_import, division
+import KratosMultiphysics
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.kratos_utilities as kratos_utilities
+import RVEAnalysis
```

Comment:
```suggestion
from KratosMultiphysics.StructuralMechanicsApplication.rve_analysis import RveAnalysis
```

Reply:
done (still calling it RVEAnalysis instead of RveAnalysis)

Reply:
fine for me, as you explained me why :+1: